.github/workflows/versionci.yml

```diff
@@ -0,0 +1,20 @@
+name: Version CI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ["3.2.0", "3.2.2", "3.3.0"]
```

Comment:
What's the rationale for testing 3.2.2 in addition to 3.2.0? Optimistically, a patch release should only contain backwards-compatible bug fixes. Also, the Ruby site says 3.2.4 is the latest stable, so maybe 3.2.2 is already outdated if we are sticking with testing all patched versions?
![Image 2024-05-24 at 4 14 58 PM](https://github.com/LaunchPadLab/decanter/assets/20426095/0b80b938-d2a8-4d7a-accb-f315edb6f9dd)